src/quacc/calculators/espresso/calculators.py

```diff
@@ -0,0 +1,127 @@
+from quacc import SETTINGS
+from quacc.calculators.espresso.core import EspressoCalculator
+from quacc.calculators.espresso.keys import (DYNMAT_KEYS, NEB_KEYS, PH_KEYS,
+                                             Q2R_KEYS, MATDYN_KEYS, PW_KEYS)
+from quacc.calculators.espresso.utils import parse_pw_preset
+
+# Mini classes that might be useful for binaries
+# that have very specific requirements
+
+class Ph(EspressoCalculator):
+    # These are not supposed to change
+    _BIN = 'ph'
+    _KEYS = PH_KEYS
+    _CMD = SETTINGS['ESPRESSO_PH_CMD']
+
+    def __init__(self, input_data):
+        self.input_data = input_data
+        super().__init__()
+
+class Neb(EspressoCalculator):
+
+    # I think it makes sense to have the defaults
+    # here. They are not supposed to change.
+    # Users can have a look at them by doing:
+    # from quacc.calculators.espresso.calculators import Neb
+    # Neb._calc_defaults without looking at the source code
```

Comment:
Hmmm! This is one I'm not 100% confident about. Perhaps it would make more sense to have as a global variable in `recipes.espresso.neb` (i.e. outside the scope of any particular function). This will still allow importing of the defaults (`from quacc.recipes.espresso.neb import _BASE_NEB_PARAMS` or something) while still making it immediately visible to users.

I have purposefully tried to keep all defaults within the recipe modules because that is often where people will immediately look for details (e.g. "oh, a relax_job... let me open that up to see what it does"). If things are _never_ supposed to be changed by the user, I care a bit less though so could be swayed. 

Regardless, I would recommend making the `num_of_images` accessible via a kwarg in the recipe (e.g. `n_images`) since that's one where users typically want to adjust things.